_config.yml

```diff
@@ -43,10 +43,18 @@ defaults:
       type: events
     values:
       layout: event
+  - scope:
+      path: ""
+      type: resources
+    values:
+      layout: resource
 
 collections:
   events:
     output: true
     permalink: /:year-:month-:day.html
   posts:
     permalink: /:year-:month-:day-:title.html
+  resources:
+    output: true
+    permalink: /:title.html
```

Comment:
I just added another commit that moves the main resources page to 
resources/index.html
and individual resources to 
resources/tonejs.html.

Are you asking for the resources to be
resources/tonejs/index.html?